env_configurations.go

```diff
@@ -0,0 +1,30 @@
+package api
+
+import (
+	"fmt"
+	"gopkg.in/nullstone-io/go-api-client.v0/response"
+	"io"
+	"io/ioutil"
+	"net/http"
+)
+
+type EnvConfigurations struct {
+	Client *Client
+}
+
+func (ec EnvConfigurations) basePath(stackId, envId int64) string {
+	return fmt.Sprintf("/orgs/%s/stacks/%d/envs/%d/runs", ec.Client.Config.OrgName, stackId, envId)
+}
+
+func (ec EnvConfigurations) Create(stackId, envId int64, file io.Reader) error {
+	raw, err := ioutil.ReadAll(file)
```

Comment:
I think the HTTP client Do accepts an io.Reader for the body.

Reply:
👍